Metchik/Screens/Home/Home/HomeView.swift

```diff
@@ -18,20 +18,20 @@ struct HomeView: View {
             headerView
             
             SearchBarView(searchText: .constant(""))
-                .padding(.horizontal,25)
             quickCategoryView
             
             contentScrollView
         }
         .offset(CGSize(width: 0,height: (offset < height) ? (offset > -1 ) ? -offset : 0  : -height))
-        .animation(.spring)
+        //.animation(.spring())
```

Comment:
remove this also

Reply:
can we use it later so late it as a comment